lib/model/query/submission-defs.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
                 .orWhereIn('form_defs.keyId', keyIds))))
           .modify(applyODataFilter(options.filter))
           .where({ 'submissions.formId': formId, 'submissions.deletedAt': null })
-          .orderBy('submissions.id', 'desc')
+          .orderBy('submissions.createdAt', 'desc')
```

Comment:
Do we want to order by `createdAt` and then also `id` so that if there are ties on `createdAt`, the order is stable? That has to be very unlikely, right?